world-core/src/router/Router.ts

```diff
@@ -42,7 +42,7 @@ export class Router {
         }
 
         catch (e) {
-            Logger.warn(`Problem emitting message '${msg.type}'`, { exception: e.toString() });
+            Logger.warn(`Problem emitting message '${msg.type}'`, { exception: e.tostring() });
```

Comment:
This is odd, it should be `.toString()` on exceptions. Was this not working for you? What was the error message? Same question for `world-server/src/network/Server.ts` change as well.

Reply:
Just an artifact of find and replace. Fixed it